### DISCUSSION TITLE
Can any one give me an advice please 🥺

### DISCUSSION THREAD

**Discussion by Lonelybarbarian0** [WerWolv/ImHex#1479] [General]
Hi I am 18 and I want to start myself onto embedded software and low level programming , I know c programming and iam currently learning c++ , I really want to learn more about Linux and stuff but I don't know where to start. And seeing your guys work it seems like you guys have a good amount of technical knowledge and if you guys can help me out it would be really great .

**Comment by WerWolv** [comment]
Hey, feel free to join our Discord server (it's linked in the Readme), there's a couple of people there working on learning similar things too :)
